api/client/swarm/join_token.go

```diff
@@ -0,0 +1,71 @@
+package swarm
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/docker/docker/api/client"
+	"github.com/docker/docker/cli"
+	"github.com/docker/engine-api/types/swarm"
+	"golang.org/x/net/context"
+)
+
+const (
+	flagRotate = "rotate"
+)
+
+func newJoinTokenCommand(dockerCli *client.DockerCli) *cobra.Command {
+	var rotate bool
+
+	cmd := &cobra.Command{
+		Use:   "join-token [--rotate] worker | manager",
```

Comment:
i wonder if it should have parentheses `(worker|manager)` (like in docopt.org) not sure what's the standard here